webpack.config.js

```diff
@@ -23,7 +23,6 @@ const nodeExternals = require('webpack-node-externals')
 const commonConfig = {
   mode: 'production',
   optimization: {
-    // Minification breaks angular.
```

Comment:
Does the removal of this comment imply that minification could be turned on? Or is that only relevant to web-apps and not applicable to desktop-apps?

Reply:
It was disabled in the first place because it was breaking angular.
Now that we no longer have angular it could be enabled.
But we'll keep it disabled for easier debugging.